packages/office-ui-fabric-react/src/components/Callout/Callout.Props.ts

```diff
@@ -44,6 +44,12 @@ export interface ICalloutProps extends React.Props<Callout | CalloutContent> {
   beakWidth?: number;
 
   /**
+   * The background color of the beak.
+   * @default #ffffff
+   */
+  beakColor?: string;
```

Comment:
This really seems like a very weird thing to me.

Wouldn't we want to define the entire background color of the callout and make the background and beak match? Why specifically the beak, why "Color" instead of "BackgroundColor"? Why default to white?